### PR TITLE
document [group],CloneFromSampler

### PR DESCRIPTION
**Pull request by ronso0**
added in https://github.com/mixxxdj/mixxx/pull/3200

### CODE REVIEW

File: source/chapters/advanced_topics.rst [hunk 1]
```diff
@@ -851,6 +851,18 @@ Any control listed above for :mixxx:cogroupref:`[ChannelN]` will work for a samp
    .. versionadded:: 2.3.0
 
 
+.. mixxx:control:: [ChannelN],CloneFromSampler
+                   [PreviewDeckN],CloneFromSampler
+                   [SamplerN],CloneFromSampler
+
+   Clone the given sampler number, copying the play state, position, rate, and key.
+
+   :range: integer
```
Comment (Holzhaus) [inline source/chapters/advanced_topics.rst:860]: Cna elaborate a bit more here, e. g.:
```suggestion
   :range: integer between 1 and :mixxx:coref:`[Master],num_samplers` (inclusive) 
```

**Review by Holzhaus — CHANGES_REQUESTED** [review]

File: source/chapters/advanced_topics.rst (hunk 1 above)
Comment (ronso0) [reply source/chapters/advanced_topics.rst]: :+1:

Comment (ronso0) [reply source/chapters/advanced_topics.rst]: added this to CloneFromDeck as well

**Review by Holzhaus — APPROVED** [review]
LGTM, thank you!

**Pull request merged**